Binary files aaa/elinks-0.4pre15/intl/.english.lng.swp and elinks-0.4pre15/intl/.english.lng.swp differ
diff -urN aaa/elinks-0.4pre15/intl/polish.lng elinks-0.4pre15/intl/polish.lng
--- aaa/elinks-0.4pre15/intl/polish.lng	Tue Aug 27 17:38:50 2002
+++ elinks-0.4pre15/intl/polish.lng	Thu Aug 29 14:15:42 2002
@@ -49,14 +49,14 @@
 T_LINUX_OR_OS2_FRAMES, "Ramki typu Linux lub OS/2",
 T_KOI8R_FRAMES, "Ramki KOI8-R",
 T_USE_11M, "U¿yj ^[[11m",
-T_UTF_8_IO, NULL,
+T_UTF_8_IO, "UTF-8 We/Wy",
 T_RESTRICT_FRAMES_IN_CP850_852, "Ogranicz ramki w cp850/852",
 T_BLOCK_CURSOR, "Kursor blokowy",
 T_COLOR, "Kolor",
 T_TERMINAL_OPTIONS, "Opcje terminala",
 T_HTTP_PROXY__HOST_PORT, "HTTP proxy (host:port)",
 T_FTP_PROXY__HOST_PORT, "FTP proxy (host:port)",
-T_NOPROXY_LIST, NULL,
+T_NOPROXY_LIST, "Domeny bezpo¶redniego dostêpu (bez proxy)",
 T_MAX_CONNECTIONS, "Maksymalna liczba po³±czeñ",
 T_MAX_CONNECTIONS_TO_ONE_HOST, "Maksymalna liczba po³±czeñ z jednym serwerem",
 T_RETRIES, "Próby",
@@ -75,8 +75,8 @@
 T_NUMBERED_LINKS, "Numerowane linki",
 T_TEXT_MARGIN, "Margines tekstu",
 T_IGNORE_CHARSET_INFO_SENT_BY_SERVER, "Ignoruj informacje o zestawie znaków wysy³ane przez serwer",
-T_USE_DOCUMENT_COLOURS, NULL,
-T_ALLOW_DARK_ON_BLACK, NULL,
+T_USE_DOCUMENT_COLOURS, "U¿ywaj kolorów okre¶lonych przez dokument",
+T_ALLOW_DARK_ON_BLACK, "Dopuszczaj u¿ycie ciemnych kolorów na czarnym tle" ,
 T_HTML_OPTIONS, "Ustawienia HTML",
 T_DEFAULT_CODEPAGE, "Domy¶lna strona kodowa",
 T_GOTO_URL, "Przejd¼ do URL-a",
@@ -118,15 +118,15 @@
 T_ENTER_URL, "Wprowad¼ URL",
 T_SAVE_URL, "Zapisz URL",
 T_DOWNLOAD, "Pobieranie",
-T_DOWNLOAD_COMPLETE, NULL,
+T_DOWNLOAD_COMPLETE, "Pobieranie zakoñczone",
 T_SAVE_TO_FILE, "Zapisz do pliku",
 T_SEARCH_FOR_TEXT, "Znajd¼ tekst",
 T_WAITING_IN_QUEUE, "Oczekuj±ce w kolejce",
 T_LOOKING_UP_HOST, "Szukam serwera",
 T_MAKING_CONNECTION, "Nawi±zywanie po³±czenia",
-T_SSL_NEGOTIATION, NULL,
+T_SSL_NEGOTIATION, "Negocjacja SSL",
 T_REQUEST_SENT, "¯±danie wys³ane",
-T_LOGGING_IN, NULL,
+T_LOGGING_IN, "Logujê siê",
 T_GETTING_HEADERS, "Pobieranie nag³ówków",
 T_SERVER_IS_PROCESSING_REQUEST, "Serwer przetwarza ¿±danie",
 T_TRANSFERRING, "Przesy³anie",
@@ -145,11 +145,11 @@
 T_CANT_GET_SOCKET_STATE, "Nie mo¿na zbadaæ stanu gniazda",
 T_BAD_HTTP_RESPONSE, "Nieprawid³owa odpowied¼ HTTP",
 T_HTTP_100, NULL,
-T_FAKE_REFERER, NULL,
-T_REFERER_NONE, NULL,
-T_REFERER_TRUE, NULL,
-T_REFERER_SAME_URL, NULL,
-T_REFERER_FAKE, NULL,
+T_FAKE_REFERER, "Sta³e odwo³anie HTTP",
+T_REFERER_NONE, "Brak odwo³ania",
+T_REFERER_TRUE, "Wy¶lij poprzedni URL jako odwo³anie (normalna operacja, NIEBEZPIECZNE)",
+T_REFERER_SAME_URL, "Wy¶lij ¿±dany URL jako odwo³anie",
+T_REFERER_FAKE, "Sta³e odwo³anie",
 T_NO_CONTENT, "Plik pusty",
 T_UNKNOWN_FILE_TYPE, "Nieznay typ pliku",
 T_ERROR_OPENING_FILE, "B³±d przy otwieraniu pliku",
@@ -170,7 +170,7 @@
 T_ELAPSED_TIME, "Up³ynê³o",
 T_ESTIMATED_TIME, "Pozosta³y czas",
 T_BACKGROUND, "W tle",
-T_BACKGROUND_NOTIFY, NULL,
+T_BACKGROUND_NOTIFY, "W tle z powiadomieniem",
 T_ABORT, "Przerwij",
 T_YES, "Tak",
 T_NO, "Nie",
@@ -188,7 +188,7 @@
 T_SAVE_IT_OR_DISPLAY_IT, "otworzyæ, zapisaæ czy wy¶wietliæ?",
 T_OPEN, "Otwórz",
 T_DO_YOU_WANT_TO_FOLLOW_REDIRECT_AND_POST_FORM_DATA_TO_URL, "Czy chcesz przekierowaæ i wys³aæ dane formularza do",
-T_USER_AGENT, NULL,
+T_USER_AGENT, "Identyfikator przegl±darki",
 T_DO_YOU_WANT_TO_POST_FORM_DATA_TO_URL, "Chcesz wys³aæ formularz do",
 T_DO_YOU_WANT_TO_REPOST_FORM_DATA_TO_URL, "Chcesz ponownie wys³aæ formularz do",
 T_WARNING, "Ostrze¿enie",
@@ -219,12 +219,12 @@
 T_ERROR_WRITING_TO_FILE, "B³±d zapisu do pliku",
 T_DISPLAY_USEMAP, "Poka¿ mapê obrazków",
 T_FOLLOW_LINK, "Wybierz link",
-T_FOLLOW_LINK_RELOAD, NULL,
+T_FOLLOW_LINK_RELOAD, "Wybierz link i prze³aduj",
 T_OPEN_IN_NEW_WINDOW, "Otwórz w nowym oknie",
 T_DOWNLOAD_LINK, "Pobierz",
 T_RESET_FORM, "Wyczy¶æ formularz",
 T_SUBMIT_FORM, "Prze¶lij formularz",
-T_SUBMIT_FORM_RELOAD, NULL,
+T_SUBMIT_FORM_RELOAD, "Prze¶lij formularz i prze³aduj",
 T_SUBMIT_FORM_AND_OPEN_IN_NEW_WINDOW, "Prze¶lij formularz i otwórz w nowym oknie",
 T_SUBMIT_FORM_AND_DOWNLOAD, "Prze¶lij formularz i ¶ci±gnij",
 T_VIEW_IMAGE, "Poka¿ obrazek",
@@ -247,14 +247,14 @@
 T_SUBMIT_TO, "przes³aæ do",
 T_POST_TO, "wys³aæ do",
 T_INFO, "Informacja",
-T_HEADER_INFO, NULL,
+T_HEADER_INFO, "Informacje o nag³ówku",
 T_YOU_ARE_NOWHERE, "Tak naprawdê to jeste¶ nigdzie!",
 T_URL, "URL",
 T_SIZE, "Rozmiar",
-T_TITLE, NULL,
-T_LAST_VISIT_TIME, NULL,
-T_LINK_TITLE, NULL,
-T_UNKNOWN, NULL,
+T_TITLE, "Tytu³",
+T_LAST_VISIT_TIME, "Data ostatniej wizyty",
+T_LINK_TITLE, "Tytu³ linku",
+T_UNKNOWN, "Nieznany",
 T_INCOMPLETE, "niekompletne",
 T_CODEPAGE, "Strona kodowa",
 T_ASSUMED, "Domy¶lna",
@@ -284,11 +284,11 @@
 T_EDIT_BOOKMARK, "Edytuj zak³adkê",
 T_DELETE_BOOKMARK, "Usuñ zak³adkê",
 T_BOOKMARK_MANAGER, "Obs³uga zak³adek",
-T_GLOBAL_HISTORY, NULL,
-T_HISTORY_MANAGER, NULL,
-T_DELETE_HISTORY_ITEM, NULL,
-T_CLEAR_GLOBAL_HISTORY, NULL,
-T_SEARCH_HISTORY, NULL,
+T_GLOBAL_HISTORY, "Historia globalna",
+T_HISTORY_MANAGER, "Historia globalna",
+T_DELETE_HISTORY_ITEM, "Usuñ pozycjê historii",
+T_CLEAR_GLOBAL_HISTORY, "Wyczy¶æ historiê globaln±",
+T_SEARCH_HISTORY, "Przeszukaj historiê",
 T_NNAME, "Nazwa",
 T_EXIT_LINKS, "Wyjd¼ z linksa",
 T_DO_YOU_REALLY_WANT_TO_EXIT_LINKS, "Czy na pewno chcesz wyj¶æ z linksa",
@@ -314,13 +314,13 @@
 T_BAD_MAILTO_URL, "B³êdny adres e-mail",
 T_BAD_TELNET_URL, "Niew³a¶ciwy adres serwera dla programu Telnet",
 T_BAD_TN3270_URL, "Niew³a¶ciwy adres serwera dla programu Tn3270",
-T_USERID, NULL,
-T_PASSWORD, NULL,
-T_ENTER_USERNAME, NULL,
-T_AUTHEN, NULL,
+T_USERID, "Nazwa u¿ytkownika",
+T_PASSWORD, "Has³o",
+T_ENTER_USERNAME, "Potrzebne uwierzytelnienie dla ",
+T_AUTHEN, "Uwierzytelnienie HTTP",
 T_AT, NULL,
-T_SSL_ERROR, NULL,
-T_NO_SSL, NULL,
+T_SSL_ERROR, "B³±d SSL",
+T_NO_SSL, "Ta wersja elinksa nie posiada wsparcia dla SSL/TSL",
 T_HK_ADD_BOOKMARK, "D",
 T_HK_BOOKMARKS, "Z",
 T_HK_GLOBAL_HISTORY, NULL,
@@ -390,18 +390,18 @@
 T_HK_FULL_SCREEN, NULL,
 T_HK_BEOS_TERMINAL, NULL,
 T_HK_NEW_WINDOW, NULL,
-T_FEATURES, NULL,
-T_LUA_ERROR, NULL,
-T_LUA_CONSOLE, NULL,
-T_ENTER_EXPRESSION, NULL,
+T_FEATURES, "Mo¿liwo¶ci",
+T_LUA_ERROR, "B³±d Lua",
+T_LUA_CONSOLE, "Konsola Lua",
+T_ENTER_EXPRESSION, "Wprowad¼ wyra¿enie",
 T_XDIALOG_TITLE, NULL,
-T_XDIALOG_FIELD, NULL,
-T_NEED_MASTER_TERMINAL, NULL,
-T_SEARCH_BOOKMARK, NULL,
-T_CLEAR, NULL,
-T_ADD_BOOKMARK_LINK, NULL,
+T_XDIALOG_FIELD, "Pole",
+T_NEED_MASTER_TERMINAL, "Mo¿esz zrobiæ to jedynie na g³ównym terminalu",
+T_SEARCH_BOOKMARK, "Przeszukaj zak³adki",
+T_CLEAR, "Wyczy¶æ",
+T_ADD_BOOKMARK_LINK, "Dodaj zak³adkê",
 T_HK_ADD_BOOKMARK_LINK, NULL,
-T_ACCEPT_LANGUAGE, NULL,
-T_ACCEPT_UI_LANGUAGE, NULL,
-T_SSL_CIPHER, NULL,
-T_ENCODING, NULL,
+T_ACCEPT_LANGUAGE, "Preferowany jêzyk (lista kodów ISO oddzielona przecinkami)",
+T_ACCEPT_UI_LANGUAGE, "Jêzyk interfejsu taki jak Zaakceptowane-Jêzyki (ma³a strata na prywatno¶ci)",
+T_SSL_CIPHER, "Szyfr SSL",
+T_ENCODING, "Kodowanie",
